### ISSUE TITLE
Sentinel 2

### ISSUE THREAD

**Issue by kangzhengfei** [kapadia/usgs#41]
I try to get meta-data by using those methods mentioned in doc http://kapadia.github.io/usgs/index.html

I use entity id from https://earthexplorer.usgs.gov/, and the data site is sentinel 2. For exemple:
``` shell
usgs metadata --node EE SENTINEL_2A L1C_T30UYV_A016779_20180908T105708
```

But  I got 
```shell
usgs.USGSError: METADATA_SCENES_INVALID: 0 of 1 scenes were found 
```

If I use example in doc, it works

**Comment by kapadia** [comment]
@kangzhengfei I just tried with your example, and another one that I obtained by using `usgs search`:

```
usgs search --node EE SENTINEL_2A --start-date 20181001 --end-date 20181002 --geojson --latitude 35.1512 --longitude -90.0302 --distance 10000
```

Unfortunately, there's nothing I can do to help out with this error. It appears that the search endpoint for Sentinel 2 operates as it should, however, the metadata endpoint results in an error. This would be an API issue to pass along to the EarthExplorer team. The error that you see raised is actually the error returned from USGS servers, I just pass it through to the end user of this module. The error indicates that it has found 1 scene, but then something occurs which prevents it from continuing the request.
